praw/models/reddit/emoji.py

```diff
@@ -0,0 +1,19 @@
+"""Provide the EMOJI class."""
```

Comment:
Another suggestion @bboe will have to comment on:

The `Emoji` class is small. I think we might be able to just put it in `subreddit.py` since it's so simple.

Reply:
subreddit.py is a pretty large file so I'd prefer Emoji to be in its own file. In fact if someone was so inclined, most of these subreddit helpers really should be in their own files, or at least in a subreddit_helpers.py file (definitely a separate PR).